src/tests/ftest/util/control_test_base.py

```diff
@@ -7,9 +7,9 @@
 
 from avocado import fail_on
 from collections import defaultdict
-from apricot import TestWithServers
-from general_utils import get_host_data
-from command_utils import CommandFailure
+from .apricot import TestWithServers
+from .general_utils import get_host_data
```

Comment:
(pylint-import-error) Unable to import 'script.general_utils'
(pylint-unused-import) Unused get_host_data imported from general_utils